app/models/bitflyer_connection.rb

```diff
@@ -38,24 +38,6 @@ def access_token_expired?
     access_expiration_time.present? && Time.now > access_expiration_time
   end
 
-  # Makes a request to the Bitflyer API to refresh the current access_token
-  def refresh_authorization!
```

Comment:
You'll need to update where this is called too

Reply:
Yes, so this currently isn't called -- except for in `sync_connection!`, which isn't called at all. It is commented out in the Payout task and will need to be enabled there. The goal is to do that work later this week, but that will require pulling that function into another service class and out of the model -- something not in scope for this PR.